### DISCUSSION TITLE
[20/01/24] - Daily Work Notes

### DISCUSSION THREAD

**Discussion by Nintails1** [Nintails-TF/ncr-agile-wombo#2] [Meeting Notes]
- We setup a Kanban board to track our project.
- Our branch structure will involve The following
   - ![image](https://github.com/Nintails-TF/ncr-agile-wombo/assets/91065019/8d4308a0-accf-462b-aacf-d9d280afa0c1)
   - This means that we have a seperate branch for each main feature, once a feature is done, we commit it to dev. At the end of the project we commit to main.
- We began to identify user stories but then realised, how many user stories do we need?
  - We got a bit stuck at this. But we have decided on 20 user stories.
- Lucca was able to start formatting and finish our issues template. #3 
- Me, Finnbar, have decided to work on some user stories.
- We are happy working through the weekend, but scheduling on the weekend is rough for everyone.
- We decided to have a smaller standup meeting with Lucca, Me and Finnbar to discuss user stories around 2pm tommorrow.
   - This is because people haven't really had time to do anything.
   - Discuss blocking, what you have done, what you are going to do, 5-10mins.
- We came up with naming conventions for branches
   -  ![image](https://github.com/Nintails-TF/ncr-agile-wombo/assets/91065019/acd67d14-eabe-4c90-aa63-f5f1cadee456)
- We narrowed down our automated testing to Jest for our back-end. We've decided that front-end testing isn't important at the moment.
- We protected the main branch, so that now before merging with main, code must be reviewed by another person and signed off.
- Lucca added a linter to the CI/CD pipeline to reject code that is not formatted correctly.

**Comment by Lucca-C** [comment]
COMMIT CONVENTIONS
- fix:
- docs:
- feat:

E.g.: 
fix: added a button

[More reading](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
